lib/Models/Subscriber.php

```diff
@@ -120,7 +123,10 @@ static function getCurrentWPUser() {
     return self::where('wp_user_id', $wp_user->ID)->findOne();
   }
 
-  static function generateToken($email = null, $length = 32) {
+  /**
+   * @deprecated Only for backward compatibility for old tokens
```

Comment:
I don't think this can be deprecated. We have to leave this here forever for compatibility reasons, don't we?

Reply:
Deprecated might not be the best word. I just want to make sure that no one will use this function with some new feature. I think we can go with comment.